### PR TITLE
PRS 13459 60

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records, owners of MY-015

### CODE REVIEW

File: new/PRS13460WalattaLibanos.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">WalattaLibanos</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="other"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
```
Comment (eu-genia) [inline new/PRS13460WalattaLibanos.xml:53]: date?

File: new/PRS13460WalattaLibanos.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                </person>
+                <listRelation>
+                    <relation name="betmas:wifeOf" active="PRS13460WalattaLibanos" passive=""/>
```
Comment (eu-genia) [inline new/PRS13460WalattaLibanos.xml:56]: passive=""?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13460WalattaLibanos.xml (hunk 2 above)
Comment (eu-genia) [reply new/PRS13460WalattaLibanos.xml:56]: still open

Comment (eu-genia) [reply new/PRS13460WalattaLibanos.xml:56]: @DenisNosnitsin1970 can you complete the record?

Comment (DenisNosnitsin1970) [reply new/PRS13460WalattaLibanos.xml]: Done.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**